mm2src/mm2_main/tests/docker_tests_main.rs

```diff
@@ -72,6 +73,7 @@ pub fn docker_tests_runner(tests: &[&TestDescAndFn]) {
         utxo_ops.wait_ready(4);
         utxo_ops1.wait_ready(4);
 
+        wait_for_geth_node_ready();
```

Comment:
This function doesn't do anything other than holding the process for a while. Any reason to not to terminate the process if the target node isn't ready?

Reply:
> Any reason to not to terminate the process if the target node isn't ready?

Well, if the node isnt ready the calls in the `init_geth_node()` function will panic and terminate the process.
This was the idea to wait and give the `init_geth_node` calls a chance not to panic if the node is not ready right away.

Reply:
Yeah, what I am asking is `wait_for_geth_node_ready` doesn't return any value, so we don't know if the node is ready or not. I was expecting to panic here if the target node isn't ready.

Reply:
added panic if node isnt ready after several attempts https://github.com/KomodoPlatform/komodo-defi-framework/pull/2100/commits/a2c47f8680d536038b2dd168f6500375efeb65b8